ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -295,6 +311,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getItemClass = (item: ManufacturedHomeSearchResultIF): string => {
+      return item.selected && !props.isReviewMode ? 'selected' : ''
+    }
+
+    const onOwnerCheckboxClick = (item: ManufacturedHomeSearchResultIF) => {
```

Comment:
not required but return types, even on `: void`, help for consistency.

Reply:
If our rules were `strict` without exceptions, the TS would be upset 😆

Reply:
updated!